### DISCUSSION TITLE
Is there any possible way to enable linux with an exploit

### DISCUSSION THREAD

**Discussion by funkymokey** [3kh0/ext-remover#534] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
And im not talking about the exploits with "bookmarklets" and all, i have a personal computer and USB drive 

Permission to call me a retard if there's nothing for this is granted.

**Comment by BroJac5246** [comment, marked as answer]
I don't think it'll work, but what happens if you go to `chrome://crostini-installer`? I get an error when I try to install Crostini, but it might be worth a try.

**Reply by funkymokey** [reply]
It says "This process may take a few minutes. Downloading the virtual machine." i think it might have worked or is that just what it says before the erorr

**Reply by qDread-dev** [reply]
> It says "This process may take a few minutes. Downloading the virtual machine." i think it might have worked or is that just what it says before the erorr

If it doesnt show an error than congrats I think you have got it

**Reply by BroJac5246** [reply]
WAIT IT WORKED!?!?!

I honestly did not think it would! For me, the error is immediate.

It really does take a few minutes, so be patient.

**Reply by funkymokey** [reply]
I had to turn off my Chromebook earlier to switch periods, and it's been stuck as Installing Linux, no other words, and the bar isn't moving at all.

**Reply by BroJac5246** [reply]
Hmm... Do you get to take your Chromebooks home? Does New Point Blank work on your school network, or just your home network? It's possible the firewall is blocking files necessary for the installation.

**Reply by funkymokey** [reply]
> Hmm... Do you get to take your Chromebooks home? Does New Point Blank work on your school network, or just your home network? It's possible the firewall is blocking files necessary for the installation.

I am allowed to take them home, I don't know how point-blank works, I'm positive its not my home internet.

**Reply by BroJac5246** [reply]
> > Hmm... Do you get to take your Chromebooks home? Does New Point Blank work on your school network, or just your home network? It's possible the firewall is blocking files necessary for the installation.
> 
> I am allowed to take them home, I don't know how point-blank works, I'm positive its not my home internet.

New Point-Blank tricks Chrome into letting you run bookmarklets that affect your blocker extension. This lets one tell the blocker to continually restart itself. While it's restarting, it can't block anything, and since it's constantly restarting, it constantly can't block anything. This lets you access any website. However, some schools' Wi-Fi networks have their own restrictions in place that will still block those websites. Your home network probably has no such restrictions, though, so you have more control at home.

If you can, I would recommend trying to install Linux _at home_ while using New Point-Blank. First, make sure you have the ability to go to any website, then try installing Linux. Note that you may have to keep pressing the button on the Point Blank Swap Launcher for the extension to stay killed.

If you aren't successful, I'll try to help you further. If you are successful, then keep in mind that you should have the extension killed whenever Linux is downloading files. And Linux downloads A LOT of files. Just keep that in mind.

**Reply by funkymokey** [reply]
Damn, now its not even telling me its installing linux, its just a error

**Reply by BroJac5246** [reply]
> Damn, now its not even telling me its installing linux, its just a error
> 

:( I didn't think it would work, but it was worth a try
